src/Notes/CouponPageMoved.php

```diff
@@ -55,11 +61,6 @@ public static function can_be_added() {
 			return false;
 		}
 
-		// If new navigation feature is enabled.
-		if ( Features::is_enabled( 'navigation' ) ) {
-			return false;
-		}
-
```

Comment:
Nice. Removed because this constraint is present already in `should_display_legacy_menu()`